data/default-config.json

```diff
@@ -1,10 +1,31 @@
 {
-  "channel": "legi-bot",
+  "channel": "legisacn",
   "cron": "*/30 * * * *",
   "embeds": false,
   "interval": 60,
-  "prefix": "!",
+  "prefix": ">",
```

Comment:
why?

Reply:
This was to avoid collision with the bot on my testbed, I mistakenly pushed this change tho. However this raise a concern that the default channel is set to a fixed one, which can be problematic. I think we should by default listen to any channel on the server until the proper channel is defined